src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -401,6 +402,35 @@ protected function getEnv($name)
         }
     }
 
+    protected function throw($message)
```

Comment:
does it actually make sense to have it in the parent class ?

Reply:
reverted that part :)